scripts/python/testbedApi.py

```diff
@@ -0,0 +1,329 @@
+# Python testing utility classes and functions
```

Comment:
Going through this file I see some overlap with the `systemtests` package that we have. In long run is it intended to replace go-based systemtests for netplugin? Or we will be maintaining both?

I have no strong opinion for go tests v/s python tests per-se but from maintenance purposes I feel it might be good to have one test-infra per project. Again totally up to you.